lldb/source/Utility/XcodeSDK.cpp

```diff
@@ -65,27 +65,27 @@ bool XcodeSDK::operator==(const XcodeSDK &other) const {
 }
 
 static XcodeSDK::Type ParseSDKName(llvm::StringRef &name) {
-  if (name.consume_front("MacOSX"))
+  if (name.consume_front_insensitive("MacOSX"))
```

Comment:
I think I would prefer us remaining strict and just fix the spelling of bridgeOS going forward.